MRISimulation/src/CoilSensitivities.jl

```diff
@@ -28,3 +29,110 @@ function birdcageSensitivity(N::Int64,
+  ];;;
+  [ [ -2.660406e+03+1.130288e+03im,	-2.684663e+03-4.547482e+03im,	-1.091036e+03+1.039176e+04im,	+3.794462e+03-3.726057e+03im,	-9.599513e+02+2.510410e+03im ];;
+  [ +2.453693e+03-1.127183e+04im,	+3.368645e+03-1.562005e+04im,	-6.638387e+03-2.965991e+03im,	+3.831320e+03+2.531652e+03im,	-2.198729e+03-5.958389e+02im ];;
+  [ +1.428178e+04+9.163301e+02im,	+1.684055e+04-6.384269e+03im,	+1.658437e+04-1.232798e+04im,	-1.406222e+04+1.113432e+04im,	+3.592359e+03-6.771321e+03im ];;
+  [ -1.660563e+03+2.839105e+03im,	-2.896235e+02+1.029472e+03im,	-3.908763e+03+3.005041e+03im,	+4.267630e+03-1.222684e+04im,	+3.162250e+03+3.059668e+03im ];;
+  [ +1.229621e+03+1.214108e+03im,	+3.580651e+03+6.040717e+02im,	+7.638826e+02+2.616962e+03im,	+4.679096e+03+3.197829e+03im,	-2.025938e+03+1.366729e+03im ];;
+  ];;;
+  [ [ +1.532993e+03+3.085543e+03im,	+1.698160e+02-2.059932e+03im,	+8.620593e+03+5.947357e+03im,	-1.818730e+03-3.486291e+03im,	+2.804918e+03+2.657346e+02im ];;
+  [ -5.363040e+03-8.659919e+03im,	-1.811624e+03-1.273996e+04im,	+5.336066e+02+4.722509e+03im,	+5.288265e+03-4.540653e+02im,	-1.550763e+03+3.172655e+03im ];;
+  [ +1.484031e+04-8.435797e+03im,	+1.909462e+04-1.153338e+04im,	+2.644743e+04+1.609464e+03im,	-1.835121e+04+1.685986e+04im,	-2.060884e+03-1.164399e+04im ];;
+  [ +3.565514e+03+4.812732e+03im,	-5.792477e+02-1.059542e+03im,	-6.948085e+03+1.922669e+03im,	-3.102303e+03-1.536498e+04im,	+8.313277e+03-1.413219e+03im ];;
+  [ +3.645370e+03+3.144318e+03im,	+5.896468e+03-1.094847e+03im,	+3.137289e+03+4.010566e+03im,	+5.041110e+03-3.595689e+03im,	+4.239519e+03+4.264271e+03im ];;
+  ];;;
+  [ [ +9.757094e+02-4.944947e+02im,	-1.479025e+03-3.433955e+03im,	+6.554174e+03-3.580007e+03im,	-5.973520e+02+2.335976e+03im,	-1.082375e+03-1.958220e+03im ];;
+  [ -2.523083e+03+1.680365e+03im,	-4.212411e+03-1.392797e+03im,	+2.731082e+03+4.483586e+03im,	-2.006126e+03-1.093595e+03im,	+1.536401e+03-1.071117e+03im ];;
+  [ -9.129410e+03-5.654936e+03im,	-6.060043e+03-2.092658e+04im,	+2.299950e+04-2.185407e+04im,	+9.382429e+03+1.937377e+04im,	-1.130825e+04+2.193837e+03im ];;
+  [ +6.269709e+02-2.393860e+03im,	-1.782260e+03-2.694656e+03im,	+2.224400e+03-3.487669e+01im,	-4.697312e+03+2.063353e+03im,	-1.975369e+03-3.797778e+03im ];;
+  [ +8.142875e+02-3.201539e+03im,	+4.320247e+01-5.839946e+03im,	+7.607719e+03-2.151906e+03im,	-1.149005e+03+1.656691e+03im,	-3.673436e+01-2.858548e+03im ];;
+  ];;]
+
+	return sens_coeff[kx, ky, c]
+end
+
+
+function xsens(c::Int64,
+               pos::Vector{Int64},
+               dims::NTuple{4, Int64})
+
+    @assert 8 >= c "Maximum number of supported coils was exceeded!"
+
+    cpos = [(pos[1] - dims[1] / 2) / (0.5 * dims[1]), (pos[2] - dims[2] / 2) / (0.5 * dims[2])]
+    val = 0 + 0im
+    coil_coeff = 5
+    shift = (coil_coeff - 1) / 2
+
+    for i in 1:coil_coeff
+        for j in 1:coil_coeff
+            val += get_sens_coeff(i, j, c) * exp(-2im * π * ((i - shift) * cpos[1] + (j - shift) * cpos[2]) / 4)
```

Comment:
```suggestion
            val += get_sens_coeff(i, j, c) * cis(-2 * π * ((i - shift) * cpos[1] + (j - shift) * cpos[2]) / 4)
```
supposedly `cis` is more efficient